### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by RonalAlexis2**
DB-37 Ticket Solution

This is a practical exercise to be clear about the concepts of stored procedures.

reinforcing topics such as entity relationship diagrams.

SQL Server syntax

applying CRUD

### CODE REVIEW

File: SQLBiblioteca.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+CREATE TABLE biblioteca (
+    id_biblioteca int identity,
+    name_biblioteca varchar(255),    
+    primary key (id_biblioteca)
+);
+
+CREATE TABLE libro (
+    id_libro int identity,
+    edicion Decimal,
+	a�o date,
```
Comment (MauroDecodificado) [inline SQLBiblioteca.sql]: Please avoid using ñ, instead of use anio

**Review by MauroDecodificado — COMMENTED** [review]
Just minor comments, please check

File: SQLBiblioteca.sql (hunk 1 above)
Comment (RonalAlexis2) [reply SQLBiblioteca.sql]: has been correct

**Review by MauroDecodificado — APPROVED** [review]
Approved and now can be merged

**Pull request merged**